lib/tools/adb-emu-commands.js

```diff
@@ -36,6 +36,25 @@ emuMethods.rotate = async function () {
   await this.adbExec(['emu', 'rotate']);
 };
 
+emuMethods.sendSMS = async function (phoneNumber, message) {
+  if (_.isUndefined(phoneNumber)) {
```

Comment:
this verification is probably no needed - you anyway check if the number matches to regexp